navigation/navigation.css

```diff
@@ -0,0 +1,37 @@
+.nav-bar{
+    background-color:var(--white);
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    padding: 1rem;
+    align-items: center;
+}
+
+.btn-nav{
+    display: flex;
+    width: 20%;
+    justify-content: space-between;
+}
+
+.btn-nav button{
+    font-size: 1.1rem;
+    border: none;
+    padding: 0.5rem 1rem 0.5rem 1rem;
```

Comment:
This can be written as `padding:0.5rem 1rem`

Reply:
Thank you for letting me know this. Will change that.